multipatch_analysis/ui/acq4_modules.py

```diff
@@ -1,9 +1,24 @@
+import acq4.pyqtgraph as pg
 from acq4.pyqtgraph.Qt import QtCore, QtGui
 from acq4.modules.Module import Module
 from acq4.Manager import getManager
+import acq4.util.DataManager as DataManager
+
+# from acq4.analysis.modules.MosaicEditor import MosaicEditor
+# from acq4.util.Canvas.items.CanvasItem import CanvasItem
+from acq4.modules.MosaicEditor import MosaicEditor
+from acq4.util.Canvas.items.MarkersCanvasItem import MarkersCanvasItem 
+from acq4.util.Canvas.items.MultiPatchLogCanvasItem import MultiPatchLogCanvasItem 
 from . import submit_expt
 from . import multipatch_nwb_viewer
-
+from .. import lims
+import os
+from acq4.util import FileLoader
```

Comment:
Looks like FileLoader and shutil imports are not used.

Reply:
shutil is commented out of use for right now. I am saving a local copy of the location json and then copying it to the incoming folder afterward. Fileloader is removed